prep-html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
Be sure to include a `<head></head>` element in your HTML document.

Reply:
![Screen Shot 2022-02-21 at 7 17 54 PM](https://user-images.githubusercontent.com/99834589/155056928-776d0297-6c1e-431c-97e7-708ff60cfe3a.png)